handleMessage.js

```diff
@@ -64,11 +64,7 @@ exports.handleMessage = function(hook_name, context, callback){
     });
   }
 
-  if(iscursorMessage === true){
-    callback([null]);
-  }else{
-    callback(true);
-  }
```

Comment:
Line 32 already checks for `iscursorMessage`, and returns if it enters the conditional:

```
   if(!iscursorMessage){	   
     callback(false);	     
     return false;	     
   }
```

Code on `else` is never reached.